src/PIL/BmpImagePlugin.py

```diff
@@ -82,34 +82,34 @@ def _bitmap(self, header=0, offset=0):
         # -------------------------------------------------- IBM OS/2 Bitmap v1
         # ----- This format has different offsets because of width/height types
         if file_info["header_size"] == 12:
-            file_info["width"] = i16(header_data[0:2])
-            file_info["height"] = i16(header_data[2:4])
-            file_info["planes"] = i16(header_data[4:6])
-            file_info["bits"] = i16(header_data[6:8])
+            file_info["width"] = i16(header_data, 0)
```

Comment:
```suggestion
            file_info["width"] = i16(header_data)
```
Zero is the default value for the offset. Throwing the idea out there of not explicitly setting the argument?

Reply:
I think specifying the offset explicitly is easier to read in this case. I think the only time I would omit the offset is when the whole value is to be converted.